backend/internal/models/book.go

```diff
@@ -28,4 +28,5 @@ type Book struct {
 	Author    string    `json:"author"`
 	Image     *string   `json:"image"`      // image link
 	CreatedAt time.Time `json:"created_at"` // Following ISO 8601
+	Languages []string  `json:"languages"`  // Array of languages (ISO 639-1)
```

Comment:
really minor but can you shift this up one line? Usually created_at fields appear at the bottom of JSON objects